recipe/meta.yaml

```diff
@@ -9,19 +9,18 @@ source:
   sha256: c319bc9d3616d1bb840469e4e40bb112772af0679c262dd6d247bacad501b0a7
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: {{ PYTHON }} -m pip install . --no-deps -vv
   entry_points:
     - depfinder = depfinder.cli:cli
 
 requirements:
   host:
-    # Python version is limited by stdlib-list.
```

Comment:
Being able to run it on Python 3.9 does not mean it can inspect Python 3.9 code without the corresponding `stdlib-list`.

Reply:
Right, I'm hoping that `stdlib-list`'s version constraints will actually make certain versions of python (eg 3.10) unsolvable.